### ISSUE TITLE
2026-01-07: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by JasonZhao47** [JasonZhao47/argus#34]
# 每日更新报告（2026-01-07）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2026-01-07 23:53:54 | Kate Cheng | [Perf][Kernels] Enable FlashInfer DeepGEMM swapAB on SM90 (for W8A8 Linear Op) (#29213) |
| 2026-01-07 22:45:49 | R3hankhan | [OpenAI] Extend VLLMValidationError to additional validation parameters (#31870) |
| 2026-01-07 21:44:36 | Cyrus Leung | [Chore] Migrate V0 attention utils (#31891) |
| 2026-01-07 21:08:29 | Jared Wen | [Refactor] GLM-ASR Modeling (#31779) |
| 2026-01-07 19:25:03 | vllmellm | [ROCm][AITER] fix wrong argument passed to  AITER `flash_attn_varlen_func` (#31880) |
| 2026-01-07 17:18:52 | Andy Liu | [Bugfix][MTP] Fix GLM4 MoE fp8 loading with MTP on (#31757) |
| 2026-01-07 17:00:16 | BlankR | [Misc] Improve error messages for unsupported types and parameters (#30593) |
| 2026-01-07 16:18:28 | maang | [Model] Cleanup: Remove redundant manual definition of `make_empty_intermediate_tensors` in GLM-4-MoE (#31869) |
| 2026-01-07 16:10:29 | sihao_li | [XPU]fallback to TRITON_ATTN on xpu when use float32 dtype (#31762) |
| 2026-01-07 16:07:16 | weiyu | [Refactor][TPU] Remove torch_xla path and use tpu-inference (#30808) |
| 2026-01-07 15:36:13 | xuebwang-amd | [Bugfix][Kernel] fix bias adding in triton kernel implemented fused moe (#31676) |
| 2026-01-07 14:55:03 | Kevin McKay | [Bugfix][Hardware][AMD] Consolidate FP8 min/max values helper function (#31106) |
| 2026-01-07 14:49:39 | ℍ𝕠𝕝𝕝𝕠𝕨 𝕄𝕒𝕟 | [BugFix] LoRA: Support loading base_layer of experts (#31104) |
| 2026-01-07 14:45:10 | tianshu-Michael-yu | [Bugfix] Fix race condition in async-scheduling for vlm model (#31841) |
| 2026-01-07 14:42:20 | tjp_zju | refactor: find_loaded_library (#31866) |
| 2026-01-07 13:31:34 | Lucas Wilkinson | [Attention][3/n] Remove usage of deprecated `seq_lens_cpu` and `num_computed_tokens_cpu` CommonAttentionMetadata properties (#31850) |
| 2026-01-07 13:04:53 | vllmellm | [ROCm][AITER] bugfix accuracy regression in ROCM_AITER_TRITON_MLA backend (#31816) |
| 2026-01-07 12:34:04 | Cyrus Leung | [Chore] Try remove `init_cached_hf_modules` (#31786) |
| 2026-01-07 12:08:47 | Jack Yang | fixed mypy warnings for files vllm/v1/attention with TEMPORARY workaround (#31465) |
| 2026-01-07 11:48:13 | Tyler Michael Smith | Change warning in get_current_vllm_config to report caller's line number (#31855) |
| 2026-01-07 11:27:40 | Cyrus Leung | [Doc] Update release docs (#31799) |
| 2026-01-07 10:09:32 | Ce Zhao | [Model] Enable LoRA support for PaliGemma (#31656) |
| 2026-01-07 10:02:42 | Yihua Cheng | [1/2][lmcache connector] clean up lmcache multi-process adapter  (#31838) |
| 2026-01-07 09:37:51 | Lucas Kabela | [Misc][BE] Type coverage for vllm/compilation [1/3] (#31554) |
| 2026-01-07 09:13:24 | vSeamar | [Frontend] Implement robust video frame recovery for corrupted videos (#29197) |
| 2026-01-07 09:12:23 | Andreas Karatzas | [ROCm][CI] Fix plugin tests (2 GPUs) failures on ROCm and removing `VLLM_FLOAT32_MATMUL_PRECISION` from all ROCm tests (#31829) |
| 2026-01-07 08:31:52 | Angela Yi | [CI] Add warmup run in test_fusion_attn (#31183) |
| 2026-01-07 07:46:56 | Cyrus Leung | [Bugfix] Handle mistral tokenizer in get_hf_processor (#31817) |
| 2026-01-07 07:23:11 | Andreas Karatzas | [ROCm][CI] Pinning timm lib version to fix ImportError in Multi-Modal Tests (Nemotron) (#31835) |
| 2026-01-07 07:21:15 | Andreas Karatzas | [ROCm][CI] Fix ModernBERT token classification test numerical accuracy on ROCm (#31820) |
| 2026-01-07 05:21:42 | Matthew Bonanni | [Spec Decode][UX] Add acceptance stats to `vllm bench serve` report (#31739) |
| 2026-01-07 04:24:19 | Elvir Crnčević | Report error log after vllm bench serve (#31808) |
| 2026-01-07 04:11:26 | Nikhil G | Fix RecursionError in MediaWithBytes unpickling (#31191) |
| 2026-01-07 03:10:18 | Li, Jiang | [Quantization][Refactor] Move CPU GPTQ kernel into MP linear (#31801) |
| 2026-01-07 02:50:43 | Charlie Fu | [ROCm][CI] Fix tests/compile unit tests (#28895) |
| 2026-01-07 02:50:37 | Benjamin Chislett | [Perf] Async Scheduling + Speculative Decoding + Structured Outputs (#29821) |
| 2026-01-07 01:57:56 | Yakine Tahtah | [Bugfix] Fix GLM-4 MoE router logits dtype for data parallel chunking (#31055) |
| 2026-01-07 01:36:24 | Masataro Asai | make 500: InternalServerError more informative (#20610) |
| 2026-01-07 01:32:55 | Ning Xie | [Log] add log about gpu worker init snapshot and requested memory (#29493) |
| 2026-01-07 01:32:46 | Vadim Gimpelson | [PERF] Speed-up of GDN attention decode part (Qwen3-Next) (#31722) |
| 2026-01-07 01:32:14 | Lucas Wilkinson | [Attention][2/n] Remove usage of deprecated `seq_lens_cpu` and `num_computed_tokens_cpu` CommonAttentionMetadata properties (#31774) |
| 2026-01-07 01:07:19 | Jinzhen Lin | [Quantization][MoE] remove unused ep logic from moe marlin (#31571) |
| 2026-01-07 00:00:40 | roikoren755 | [NemotronH] Use ReplicatedLinear for fc1_latent_proj (#31807) |
